stdlib/messaging/kafka/src/main/java/org/ballerinalang/messaging/kafka/utils/KafkaConstants.java

```diff
@@ -45,10 +45,10 @@ private KafkaConstants() {
             KAFKA_PACKAGE_NAME);
 
     // Kafka log messages
-    public static final String SERVICE_STARTED = "[ballerina/kafka] started kafka listener";
-    public static final String SERVICE_STOPPED = "[ballerina/kafka] stopped kafka listener";
-    public static final String KAFKA_SERVERS = "[ballerina/kafka] kafka servers: ";
-    public static final String SUBSCRIBED_TOPICS = "[ballerina/kafka] subscribed topics: ";
+    public static final String SERVICE_STARTED = "[ballerina/kafka] Started kafka listener ";
```

Comment:
Starting letter of the log message should be in lower case, as that is the practice we use.